src/components/transactions/transactionsFormatters.test.ts

```diff
@@ -29,3 +29,31 @@ describe("Given a call to format Transaction Number function", () => {
         });
     });
 });
+
+describe("Given a call to the format Transaction Price function", () => {
+    const amount = 50;
+    let currency: string;
+
+    describe("when called with GBP as a currency", () => {
+        beforeEach(() => {
+            currency = "GBP";
+        });
+
+        it(`it returns "£${amount}"`, () => {
+            expect(formatTransactionPrice(amount, currency)).toBe(`£${amount}`);

```

Comment:
A minor thing. You have basically the same code here in the test as you do in the deployed code so all you are really testing here is that the same code does the same thing when copied to a different place.
```suggestion
            expect(formatTransactionPrice(amount, currency)).toBe("£50");

```
Also change the amount to have .00 to make the test more realistic.